swift/Samples/ModalContainer/ModalContainer.podspec

```diff
@@ -0,0 +1,20 @@
+Pod::Spec.new do |s|
+  s.name         = 'ModalContainer'
+  s.version      = '1.0.0.LOCAL'
+  s.summary      = 'See the README.'
+  s.homepage     = 'https://www.github.com/square/workflow'
+  s.license      = 'Apache License, Version 2.0'
+  s.author       = 'Square'
+  s.source       = { git: 'Not Published', tag: "podify/#{s.version}" }
+
+  # 1.7 is needed for `swift_versions` support
+  s.cocoapods_version = '>= 1.7.0.beta.1'
+
+  s.swift_versions = ['5.0']
+  s.ios.deployment_target = '9.3'
```

Comment:
Have we dropped 9 in Workflow as we have in SPOS? If so this can be 10

Reply:
We haven't bumped [Workflow](https://github.com/square/workflow/blob/0f0f7d7c9593ee07cb3f2129b3b18a5ea760e94a/Workflow.podspec#L14) yet. We probably should, though.

Reply:
Yes, let's bump it up in a different PR.